### PR TITLE
blog: add models of society post

### PR DESCRIPTION
**Pull request by jalehman**
Adds [Tyler's Models of Society](https://docs.google.com/document/d/1hSJlMCtscaSTJ6IFXkcZht0YqkOJcHfMYWk29o2kwmU/edit?ts=5f9c72a3#) essay.

### CODE REVIEW

**Comment by jalehman** [comment]
FYI @tylershuster

File: content/blog/20201119-models-of-society.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
++++
+title = "Models of Society"
+description = ""
+date = 2020-11-19
+[extra]
+description = ""
```
Comment (matildepark) [inline content/blog/20201119-models-of-society.md]: Cut

File: content/blog/20201119-models-of-society.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
++++
+title = "Models of Society"
+description = ""
```
Comment (matildepark) [inline content/blog/20201119-models-of-society.md]: Needs a line.

File: content/blog/20201119-models-of-society.md (hunk 1 above)
Comment (jalehman) [inline content/blog/20201119-models-of-society.md]: What should go here @tylershuster ?

Comment (tylershuster) [reply content/blog/20201119-models-of-society.md]: Where does the description surface?

**Comment by urcades** [comment]
Here: https://urbit.org/blog/

—
~fabled-faster
https://urbit.org/

File: content/blog/20201119-models-of-society.md (hunk 1 above)
Comment (matildepark) [reply content/blog/20201119-models-of-society.md]: Nothing should. You don’t put it in extra.

**Comment by urcades** [comment]
My approach is generally very rough, but I tend to take a snippet of the
blog and reuse it for the description.

—
~fabled-faster
https://urbit.org/

**Comment by tylershuster** [comment]
Conversations compose society. What composes conversation — How do we digitize it in a way that enhances society without imposing upon it? How do we form this new medium, both to facilitate natural human behavior and to inspire the best of it?￼￼

**Comment by tylershuster** [comment]
Looks like the em dash doesn't render quite right. I *guess* you can use a hyphen there 

image

File: content/blog/20201119-models-of-society.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+date = 2020-11-19
+[extra]
+author = "Tyler Shuster"
+ship = "~radbur-sivmus"
+image = "https://media.urbit.org/site/posts/essays/cochno-stone.png"
++++
+
+<br>
+
+![Ludovic Maclellan Mann examining the Cochno Stone](https://media.urbit.org/site/posts/essays/cochno-stone.png)
```
Comment (tylershuster) [inline content/blog/20201119-models-of-society.md]: Can we move this back under "Modes of Messaging" ? I know it's not exactly convention

**Pull request merged**